spk/syncthing-inotify/src/dsm-control.sh

```diff
@@ -4,23 +4,28 @@
 PACKAGE="syncthing-inotify"
 DNAME="Syncthing-Inotify"
 
-# Others
-INSTALL_DIR="/usr/local/${PACKAGE}"
+# Syncthing folders
 SYNCTHING_DIR="/usr/local/syncthing"
+SYNCTHING_CONFIG_DIR="${SYNCTHING_DIR}/var"
+
+# Syncthing-Inotify reads Syncthing's config and uses its busybox and user account
 PATH="${SYNCTHING_DIR}/bin:${PATH}"
 USER="syncthing"
+INOTIFY_OPTIONS="-home=${SYNCTHING_CONFIG_DIR}"
```

Comment:
No, there are no package files shared between them. Inotify by design has to read some values from syncthing config (API key, port) to communicate with it. Or they could be passed through command line which again means that the config has to be read.